cardano-lib/mainnet/conway-genesis.json

```
@@ -0,0 +1,303 @@
+    4,
+    16000,
+    100,
+    16000,
+    100,
+    16000,
+    100,
+    16000,
+    100,
+    16000,
+    100,
+    16000,
+    100,
+    100,
+    100,
+    16000,
+    100,
+    94375,
+    32,
+    132994,
+    32,
+    61462,
+    4,
+    72010,
+    178,
+    0,
+    1,
+    22151,
+    32,
+    91189,
+    769,
+    4,
+    2,
+    85848,
+    123203,
+    7305,
+    -900,
+    1716,
+    549,
+    57,
+    85848,
+    0,
+    1,
+    1,
+    1000,
+    42921,
+    4,
+    2,
+    24548,
+    29498,
+    38,
+    1,
+    898148,
+    27279,
+    1,
+    51775,
+    558,
+    1,
+    39184,
+    1000,
+    60594,
+    1,
+    141895,
+    32,
+    83150,
+    32,
+    15299,
+    32,
+    76049,
+    1,
+    13169,
+    4,
+    22100,
+    10,
+    28999,
+    74,
+    1,
+    28999,
+    74,
+    1,
+    43285,
+    552,
+    1,
+    44749,
+    541,
+    1,
+    33852,
+    32,
+    68246,
+    32,
+    72362,
+    32,
+    7243,
+    32,
+    7391,
+    32,
+    11546,
+    32,
+    85848,
+    123203,
+    7305,
+    -900,
+    1716,
+    549,
+    57,
+    85848,
+    0,
+    1,
+    90434,
+    519,
+    0,
+    1,
+    74433,
+    32,
+    85848,
+    123203,
+    7305,
+    -900,
+    1716,
+    549,
+    57,
+    85848,
+    0,
+    1,
+    1,
+    85848,
+    123203,
+    7305,
+    -900,
+    1716,
+    549,
+    57,
+    85848,
+    0,
+    1,
+    955506,
+    213312,
+    0,
+    2,
+    270652,
+    22588,
+    4,
+    1457325,
+    64566,
+    4,
+    20467,
+    1,
+    4,
+    0,
+    141992,
+    32,
+    100788,
+    420,
+    1,
+    1,
+    81663,
+    32,
+    59498,
+    32,
+    20142,
+    32,
+    24588,
+    32,
+    20744,
+    32,
+    25933,
+    32,
+    24623,
+    32,
+    43053543,
+    10,
+    53384111,
+    14333,
+    10,
+    43574283,
+    26308,
+    10,
+    16000,
+    100,
+    16000,
+    100,
+    962335,
+    18,
+    2780678,
+    6,
+    442008,
+    1,
+    52538055,
+    3756,
+    18,
+    267929,
+    18,
+    76433006,
+    8868,
+    18,
+    52948122,
+    18,
+    1995836,
+    36,
+    3227919,
+    12,
+    901022,
+    1,
+    166917843,
+    4307,
+    36,
+    284546,
+    36,
+    158221314,
+    26549,
+    36,
+    74698472,
+    36,
+    333849714,
+    1,
+    254006273,
+    72,
+    2174038,
+    72,
+    2261318,
+    64571,
+    4,
+    207616,
+    8310,
+    4,
+    1293828,
+    28716,
+    63,
+    0,
+    1,
+    1006041,
+    43623,
+    251,
+    0,
+    1
+  ],
+  "constitution": {
+      "anchor": {
+          "dataHash": "ca41a91f399259bcefe57f9858e91f6d00e1a38d6d9c63d4052914ea7bd70cb2",
+          "url": "ipfs://QmVyzwKe8NmswkVVn2LKCorFEctKCxeBApA6d89HKWd3qH"
+      },
+      "script": "fa24fb305126805cf2164c161d852a0e7330cf988f1fe558cf7d4a64"
+  },
+  "committee": {
+    "members": {
+        "keyHash-df0e83bde65416dade5b1f97e7f115cc1ff999550ad968850783fe50": 580,
+        "keyHash-b6012034ba0a7e4afbbf2c7a1432f8824aee5299a48e38e41a952686": 580,
```

```suggestion
        "scriptHash-b6012034ba0a7e4afbbf2c7a1432f8824aee5299a48e38e41a952686": 580,
```

does it matter if these are correctly labelled as keyHashes or scriptHashes?